core/templates/dev/head/pages/exploration_player/AnswerClassificationServiceSpec.js

```diff
@@ -336,28 +349,18 @@ describe('Answer classification service with string classifier enabled',
       }
     };
 
-    it('should query the backend if no answer group matches and interaction ' +
-       'is trainable', function() {
-      var backendClassifiedOutcome = {
-        outcome: 'outcome',
-        answer_group_index: 0,
-        rule_spec_index: 0,
-        classification_categorization: EXPLICIT_CLASSIFICATION
-      };
+    it('should query the prediction service if no answer group matches and ' +
```

Comment:
What does this test do exactly? I am unable to understand. It says that 'it should query prediction service' but you are still comparing it with default outcome classification result. And nothing else indicates that prediction service has been called or something?